src/acl/acl.ts

```diff
@@ -197,19 +194,18 @@ export function getResourceAcl(
 }
 
 /**
- * Verify whether a fallback ACL was found for the given Resource.
- *
- * A Resource fetched with its ACL (e.g. using [[getSolidDatasetWithAcl]]) _might_ have a fallback ACL
- * attached, but we cannot be sure: the currently authenticated user (if any) might not have Control
- * access to one of the fetched Resource's Containers.
+ * Verify whether the given Resource has a fallback ACL attached.
  *
+ * A Resource returned using [[getSolidDatasetWithAcl]], i.e., the returned `dataset,
+ * has a fallback ACL attached if the current user
+ * has Control access to the Resource's Container from which the Resource inherits its ACL.
```

Comment:
Couple of questions since I don't think the above captures the situation completely.

1. If a Resource has its own resourceAcl and its container has also set the fallbackAcl, getSolidDatasetWithAcl would return both resourceAcl and fallbackAcl? 
If so, I should clarify "from which the Resource inherits its ACL if the Resource does not have its own resourceAcl" or something like that.

2. Also, if a resource is using its container's fallbackAcl, but the user calling getSolidDatasetWithAcl has Control access to the container's container's fallbackAcl - what is returned in the fallbackAcl?

Reply:
1. Yes, if both ACLs exist and are accessible for a given resource, both are returned by `getSolidDatasetWithAcl`.
2. The fallback ACL is the ACL of the first container `C` in the hierarchy that has a `default` ACL, so the `default` ACLs of  containers parent of `C` don't apply to children resources of `C`.

Reply:
Got it -  getSolidDatasetWithAcl will return not have the fallbackAcl if called by someone w/o control access to container C but only to the parents of C.